README.md

```diff
@@ -13,3 +13,29 @@ By using the https://github.com/akeneo-labs/DataGeneratorBundle
 
 Please use the relevant branch when using with Akeneo PIM (catalog 1.4 = pim 1.4)
 
+## How to install a catalog
+
+Create a symbolik link to the fixtures
+
+```
+ln -s /<path_to_catalog>/scalability/<catalog_size>/fixtures/ /<path_to_ee>/src/PimEnterprise/Bundle/InstallerBundle/Resources/fixtures/<catalog_size>
+```
```

Comment:
in some case as the large catalog we have to download from google drive for now. we'll try to provide catalogs on another service maybe a catalogs.akeneo.com, @BitOne what do you think about this service because the google drive is available only for us ?